src/main/resources/view/TravelPlanObjectListPanel.fxml

```diff
@@ -5,18 +5,18 @@
 <?import javafx.scene.control.TabPane?>
 <?import javafx.scene.layout.VBox?>
 
- <TabPane VBox.vgrow="ALWAYS" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1">
-      <Tab closable="false" text="Activities ">
+ <TabPane VBox.vgrow="ALWAYS" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1" fx:id="pane">
+      <Tab closable="false" text="Activities" fx:id="0">
```

Comment:
I think I added a space after Activities on purpose previously, so that the tabs are more spaced out.

Reply:
resolved.